README.md

```diff
@@ -16,7 +19,7 @@ There are two groups of target users:
    - If the analyzer author has not provided a SonarQube plugin for their analyzer then users will be able to generate a plugin from an analyzer NuGet package, although they won't be able to provide such rich metadata.
 
 ### Current status
-The generated jar file is not currently sufficient on its own to upload Roslyn issues. The complete solution requires the [C# plugin v4.5](http://docs.sonarqube.org/display/PLUG/C%23+Plugin) or higher and the [MSBuild SonarQube Scanner v2.0](http://docs.sonarqube.org/display/SONAR/Analyzing+with+SonarQube+Scanner+for+MSBuild) or higher.
+The generated jar file is not  sufficient on its own to upload Roslyn issues. The complete solution requires the [C# plugin v4.5](http://docs.sonarqube.org/display/PLUG/C%23+Plugin) or higher and the [MSBuild SonarQube Scanner v2.0](http://docs.sonarqube.org/display/SONAR/Analyzing+with+SonarQube+Scanner+for+MSBuild) or higher. See [this](https://blogs.msdn.microsoft.com/visualstudioalm/2016/02/18/sonarqube-scanner-for-msbuild-v2-0-released-support-for-third-party-roslyn-analyzers/) blog post for more information on how the various components cooperate to handle issues from Roslyn analyzers.
```

Comment:
@duncanpMS imo this sounds a bit strange `Current status: The generated jar file is not  sufficient on its own to upload Roslyn issues.`

My first impression when I read just that is that this thing isn't ready to be used yet.